join-statements/joinstatements.md

```diff
@@ -0,0 +1,423 @@
+1) To find data held in two separate data tables, we could do separate searches on each table, _or_ we could use a `JOIN` statement in order to call different data from different tables into a query.
+
+2) Assuming 2 tables, A and B:
+
+**CROSS JOIN** - it shows all possible combinations between rows from table A and rows from table B, so a result from the query would be <# of A rows> x <# of B rows>
+
+A real world example would be if I had a table with fruit juices, and another table with entrees at a high school cafeteria. A CROSS JOIN would allow me to compare all the possible combinations of meals for the high schoolers.
```

Comment:
why would you want to know all the possible combinations of meals for high schoolers?

Reply:
Add description of the real world example, and use "grandma" terms.

Reply:
No SQL

Reply:
done

Reply:
checked